.github/bors.toml

```diff
@@ -0,0 +1,9 @@
+status = [
+    "test (ubuntu-latest, stable)",
+    "test (macos-latest, stable)",
+    "test (windows-latest, stable)",
+    "rustfmt",
+    "clippy",
+    "continuous-integration/appveyor/branch"
```

Comment:
We don't need "continuous-integration/appveyor/branch". There are some CI checks in that config we can migrate later.

Reply:
Done!